modules/core/src/test/java/org/apache/ignite/internal/lang/IgniteExceptionMapperUtilTest.java

```diff
@@ -203,5 +203,12 @@ public static class CustomNoMappingException extends IgniteInternalException {
         public CustomNoMappingException() {
             super(INTERNAL_ERR, "Test internal exception [err=no mapping]");
         }
+
+        /**
+         * Creates a new instance of CustomNoMappingException with given code.
+         */
+        public CustomNoMappingException(int code) {
```

Comment:
it seems this constructor is no longer used

Reply:
yep